.hashibot.hcl

```diff
@@ -0,0 +1,35 @@
+behavior "regexp_issue_labeler" "panic_label" {
+    regexp = "panic:"
+    labels = ["crash", "bug"]
+}
+
+behavior "regexp_issue_notifier" "panic_notify" {
+    regexp = "panic:"
+    slack_channel = env.TERRAFORM_SLACK_CHANNEL
+    message = "Panic report! https://github.com/${var.repository}/issues/${var.issue_number} has a panic in it."
+}
+
+behavior "remove_labels_on_reply" "remove_stale" {
+    labels = ["waiting-reponse", "stale"]
+    only_non_maintainers = true
+}
+
+poll "label_issue_migrater" "provider_migrater" {
+    new_owner = env.PROVIDERS_OWNER
+    repo_prefix = "terraform-provider-"
+    label_prefix = "provider/"
+    excluded_label_prefixes = ["backend/", "provisioner/"]
+    excluded_labels = ["build", "cli", "config", "core", "new-provider", "new-provisioner", "new-remote-state", "provider/terraform"]
+    aliases = {
+        "provider/google-cloud" = "provider/google"
+        "provider/influx" = "provider/influxdb"
+        "provider/vcloud" = "provider/vcd"
+    }
+    issue_header = <<-EOF
+    _This issue was originally opened by ${var.user} as ${var.repository}#${var.issue_number}. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._
```

Comment:
Addressing this in a separate change is fine if you'd rather keep this focused on behavior-parity, but I think "Upcoming Provider Changes in 0.10" is old enough now that we should probably find a different wording for this that talks about it more as "just the way things are" rather than as a change to how things used to be.

Reply:
yeah parity is the current goal but feel free to customize away in follow ups with the power of HCL2 and hashibot 😄